Makefile

```diff
@@ -1,64 +1,63 @@
 VIRTUAL_ENV ?= venv
 NODE_BIN = node_modules/.bin
 
+.PHONY: all
 all: help
 
 .PHONY: help
 help:
 	@echo Beteiligung.in development tools
 	@echo
 	@echo It will either use an exisiting virtualenv if it was entered
-	@echo before or create a new one in the .env subdirectory.
+	@echo before or create a new one in the venv subdirectory.
 	@echo
 	@echo usage:
 	@echo
 	@echo "  make install         -- install dev setup"
-	@echo "  make lint            -- lint all project files"
-	@echo "  make lint-quick      -- lint all files staged in git"
+	@echo "  make clean           -- delete node modules and venv"
 	@echo "  make server          -- start a dev server"
 	@echo "  make watch           -- start a dev server and rebuild js and css files on changes"
+	@echo "  make background      -- start a dev server, rebuild js and css files on changes, and start background processes"
 	@echo "  make test            -- run all test cases with pytest"
-	@echo "  make makemessages    -- create new po files from the source"
-	@echo "  make compilemessages -- create new mo files from the translated po files"
+	@echo "  make test-lastfailed -- run test that failed last"
+	@echo "  make test-clean      -- test on new database"
+	@echo "  make coverage        -- write coverage report to dir htmlcov"
+	@echo "  make lint            -- lint all project files"
+	@echo "  make lint-quick      -- lint all files staged in git"
+	@echo "  make po              -- create new po files from the source"
+	@echo "  make mo              -- create new mo files from the translated po files"
 	@echo "  make release         -- build everything required for a release"
 	@echo
 
 .PHONY: install
 install:
 	npm install --no-save
+	npm run build
 	if [ ! -f $(VIRTUAL_ENV)/bin/python3 ]; then python3 -m venv $(VIRTUAL_ENV); fi
-	$(VIRTUAL_ENV)/bin/python3 -m pip install -r requirements/dev.txt
+	$(VIRTUAL_ENV)/bin/python3 -m pip install --upgrade -r requirements/dev.txt
```

Comment:
Hmmm, explain why.
From my understanding it should not make any difference. The install section should run on a "clean" distribution where we have no pip modules and since in the requirements the specified mocules should be all pinned down, upgrade should not make any difference.

Reply:
Yes, but installing a new version of a4 based on a branch worked in opin and ae, and not in meinberlin and in bet.in. So, I wildly guessed, that this might be the reason.

Reply:
Ok, then only leave it there if you have hard evidence that this workarounds this problem, otherwise change it :-)

Reply:
Ha! It works like that!